src/shared/components/query/GenesetsVolcanoSelector.tsx

```diff
@@ -87,6 +74,46 @@ export default class GenesetsVolcanoSelector extends QueryStoreComponent<
         );
     }
 
+    // assumes that values are symmetrically centered around 0
+    @computed get xAxisTickSize() {
+        if (this.props.plotData && this.props.plotData.length > 0) {
```

Comment:
this kinda looks testworthy, but since this is really a hyve feature, you can decide